lib/type-registry.js

```diff
@@ -32,6 +32,10 @@ TypeRegistry.prototype.registerType = function(typeName, converter) {
 
   typeName = typeName.toLowerCase();
 
+  if (typeName === 'file') {
+    throw new Error('Cannot override built-in "file" type.');
```

Comment:
Also, I don't see it being caught anywhere. So, I am assuming that it will bubble up and eventually handled. Correct?